src/main.rs

```diff
@@ -23,6 +23,14 @@ enum Command {
         /// Pull request to review (eg. `danobi/prr/24`)
         pr: String,
     },
+    /// Get a pull request and open it in $EDITOR
```

Comment:
Rather than a new subcommand to get + open, how about making prr-edit only open a review file if it exists (so no implicit prr-get). And also add a --open flag to prr-get?